tests/robot-tests/README.md

```diff
@@ -0,0 +1,43 @@
+# What is this?
+
+This was created in order to help testers get started quickly on a new project, and give new testers an idea of how to approach a new project. It gives you several things:
+
+* A test runner script
+   * Headless testing
+   * Profiling
+   * Happypath
+   * Running test suites in parallel
+* Useful RF keywords
+   * Example python library of keywords
+   * Example robot framework library of keywords
+
+# What do I need to install
+
+* Python 3
+* Pip 3
+* Pipenv
+
+Then in the test directory run
+
+```
+pipenv install
```

Comment:
I think this is asking to be Dockerised! 

Might want to look into the feasibility of this, this seems like it would be a good starting point:

https://hub.docker.com/r/ppodgorsek/robot-framework/